src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/DbCommandIntegration.cs

```diff
@@ -30,7 +30,14 @@ public static class DbCommandIntegration
         /// <param name="moduleVersionPtr">A pointer to the module version GUID.</param>
         /// <returns>The value returned by the instrumented method.</returns>
         [InterceptMethod(
-            TargetAssemblies = new[] { AdoNetConstants.AssemblyNames.SystemData, AdoNetConstants.AssemblyNames.SystemDataCommon },
+            TargetAssemblies = new[] { AdoNetConstants.AssemblyNames.SystemData },
```

Comment:
just a nit, but for readability we can use this for single assemblies:
```suggestion
            TargetAssembly = AdoNetConstants.AssemblyNames.SystemData,
```

level of care: low

Reply:
If you don't mind I'll do a separate PR to refactor this since there are some other unrelated integrations that could also be fixed at the same time

Reply:
Yeah, of course!